packages/components/src/components/popover/popover.jsx

```diff
@@ -1,34 +1,26 @@
-import classNames      from 'classnames';
-import PropTypes       from 'prop-types';
-import React           from 'react';
-import PopoverBubble   from './popover-bubble.jsx';
-import IconInfoOutline from '../icon-info-outline.jsx';
-import IconQuestion    from '../icon-question.jsx';
-import IconRedDot      from '../icon-red-dot.jsx';
+import classNames           from 'classnames';
+import PropTypes            from 'prop-types';
+import React                from 'react';
+import {
+    default as TinyPopover, // eslint-disable-line import/no-named-default
```

Comment:
`import TinyPopover, { ArrowContainer } from 'react-tiny-popover';`  doesn't work? 
This shouldn't violate the eslint rule.

Reply:
Seems to work, updated @zaki-binary